applications/tari_indexer_web_ui/src/routes/Substates/Substates.tsx

```
@@ -202,12 +195,16 @@ function SubstatesLayout() {
               </TableRow>
             </TableHead>
             <TableBody>
-              {substates.map((row: any) => (
+              {substates.map((row) => (
                 <TableRow
                   sx={{ "&:last-child td, &:last-child th": { border: 0 } }}
                 >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)